core/src/com/mygdx/insanerpg/item/Item.java

```diff
@@ -0,0 +1,10 @@
+package com.mygdx.insanerpg.item;
```

Comment:
Look at allens solution, should be under models.

Reply:
added model package to envelope items package